packages/angular/test/errorhandler.test.ts

```diff
@@ -79,6 +99,157 @@ describe('SentryErrorHandler', () => {
     expect(captureExceptionSpy).toHaveBeenCalledWith(err, expect.any(Function));
   });
 
+  it('handleError method extracts a non-empty Error', () => {
```

Comment:
I know, this predates your changes but would you mind adding a nested describe around all "handleError method" tests? Then we could extract out the subject and stick to describe/it semantics:

I'm thinking of something like this:
```suggestion
  describe('handleError method', () => {
    it('extracts a non-empty Error', () => {
```

If you don't have the time, nevermind and feel free to ignore this comment :)

Reply:
Done!